roles/power_aix_bootstrap/tasks/python_install.yml

```diff
@@ -1,33 +1,34 @@
 # Copyright (c) IBM Corporation 2020
 ---
 # CHECK for Yum on inventory host
-  - name: Check for existence of yum
-    raw: "rpm -q --quiet yum && echo true || echo false"
-    register: yum_exists
+- name: Check for existence of yum
+  raw: "rpm -q --quiet yum && echo true || echo false"
+  register: yum_exists
+  changed_when: false
 
-  - name: End play if inventory host doesn't have yum installed
-    meta: end_host
-    when: yum_exists.stdout is search("false")
+- name: End play if inventory host doesn't have yum installed
+  meta: end_host
+  when: yum_exists.stdout is search("false")
 
 # EXPAND target paths
-  - name: Expand /var target directory (+100M)
-    ibm.power_aix.filesystem:
-      filesystem: /var
-      attributes: size=+100M
-      state: present
+- name: Expand /var target directory (+100M)
+  ibm.power_aix.filesystem:
+    filesystem: /var
+    attributes: size=+100M
+    state: present
 
-  - name: Expand /opt target directory (+400M)
-    ibm.power_aix.filesystem:
-      filesystem: /opt
-      attributes: size=+400M
-      state: present
+- name: Expand /opt target directory (+400M)
+  ibm.power_aix.filesystem:
+    filesystem: /opt
+    attributes: size=+400M
+    state: present
 
 # INSTALL using yum
-  - name: Install/ Update python and requisite rpms
-    yum:
-      name: python
-      state: latest
+- name: Install/ Update python and requisite rpms
+  yum:
+    name: python
+    state: present
```

Comment:
Why changed this to present as the previous one was set to latest. If package is present, it will be updated to the latest.

Reply:
the role linting CI flags this and recommends to use `present` instead of `latest`.

i think the rationale for this is `present` is used, then if something was already installed then yum module will do nothing else.
if `latest` is used, then in addition to checking if something is already installed it will also update it to the latest version if one is available.
the whole point in this is that if something was already installed and is working, then it should be kept as is unless explicitly specified to update. the reason for it is if the `latest` version has for some reason breaks functionality for anything that depends on it or any of the dependency is not compatible anymore, then it will affect the system.
so that is why they recommend using `present` if it is not already installed.

Reply:
Is there a way to ignore this error? In the current dnf bootstrap is require to update the package if it is present. And the yum module is allowing to have this functionality in there.

Reply:
updated in a different PR